openreview/conference/invitation.py

```diff
@@ -782,9 +782,101 @@ def __init__(self, conference, note):
             reply = reply
         )
 
+class RebuttalInvitation(openreview.Invitation):
+
+    def __init__(self, conference):
+        review_rebuttal_stage = conference.review_rebuttal_stage
+        content = {
+            'rebuttal': {
+                'order': 1,
+                'value-regex': '[\\S\\s]{0,2500}',
```

Comment:
If it is easy to do, it would be great if we can have this char limit as a config parameter. 
Since we are allowing only 1 rebuttal per review, authors may need more than 2500 chars.

Reply:
I think this is the length requested by ECCV program chairs, if the user wants to change the configuration then I think we should follow the same strategy than other stages and use `additional_fields` parameter.